packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -44,6 +44,15 @@
   }
 }
 
+.mdc-chip--exit {
+  transition:
+    opacity 75ms $mdc-animation-standard-curve-timing-function,
+    width 150ms $mdc-animation-deceleration-curve-timing-function,
```

Comment:
Does this need to be width/padding/margin or can we do a scale transform instead, to take advantage of GPU compositing and eliminate potential for text truncation/wrapping issues?

I'm also a tad confused why we have the non-opacity transitions taking longer than the opacity (i.e. the element will already be invisible at that point).

Reply:
Discussed offline - transitioning the transform would be extremely complicated due to chips being wrapped to the next line.